inst/include/RcppArmadilloAs.h

```diff
@@ -403,14 +403,61 @@ namespace traits {
                     res = symmatl(res);
                 }
             }
+            else if (type == "pMatrix") {
+                IntegerVector i(ncol);
+                IntegerVector p(ncol+1);
```

Comment:
Wait, I think the same problem might also exist in ddiMatrix. I am thinking of another solution.